do nothing
[Modelarts Service Log]user: uid=1101(work) gid=1101(work) groups=1101(work)
[Modelarts Service Log]pwd: /home/work
[Modelarts Service Log]app_url: s3://resnext-tk/MA-ResNeXt-in-tensorflow-master-11-03-14-34/code/
[Modelarts Service Log]boot_file: code/cifar10_train.py
[Modelarts Service Log]log_url: /tmp/log/MA-ResNeXt-in-tensorflow-master-11-03-14-34.log
[Modelarts Service Log]command: code/cifar10_train.py --data_url=s3://resnext-tk/ResNeXt-in-tensorflow-master/cifar10_data/cifar-10-batches-py/ --train_url=s3://resnext-tk/MA-ResNeXt-in-tensorflow-master-11-03-14-34/output/V0013/ --num_gpus=1
[Modelarts Service Log]MODELARTS_IPOIB_DEVICE: 
[Modelarts Service Log]dependencies_file_dir: /home/work/user-job-dir/code
[Modelarts Service Log][modelarts_create_log] modelarts-pipe found
[Modelarts Service Log]handle inputs of training job
INFO:root:Using MoXing-v1.17.3-4c644d5b
INFO:root:Using OBS-Python-SDK-3.20.7
[ModelArts Service Log][INFO][2020/11/06 16:38:30]: env MA_INPUTS is not found, skip the inputs handler
INFO:root:Using MoXing-v1.17.3-4c644d5b
INFO:root:Using OBS-Python-SDK-3.20.7
[ModelArts Service Log]2020-11-06 16:38:31,595 - modelarts-downloader.py[line:619] - INFO: Main: modelarts-downloader starting with Namespace(dst='./', recursive=True, skip_creating_dir=False, src='s3://resnext-tk/MA-ResNeXt-in-tensorflow-master-11-03-14-34/code/', trace=False, type='common', verbose=False)
mkdir: cannot create directory ‘logs’: File exists
[ModelArts Service Log]modelarts-pipe: will create log file /tmp/log/MA-ResNeXt-in-tensorflow-master-11-03-14-34.log
[ModelArts Service Log]modelarts-pipe: will create log file /tmp/log/MA-ResNeXt-in-tensorflow-master-11-03-14-34.log
[ModelArts Service Log]modelarts-pipe: will write log file /tmp/log/MA-ResNeXt-in-tensorflow-master-11-03-14-34.log
[ModelArts Service Log]modelarts-pipe: param for max log length: 1073741824
[ModelArts Service Log]modelarts-pipe: param for whether exit on overflow: 0
/home/work/user-job-dir
[Modelarts Service Log][modelarts_logger] modelarts-pipe found
WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

Successfully downloaded cifar-10-python.tar.gz 170498071 bytes.
['cifar10_data/cifar-10-batches-py/data_batch_1', 'cifar10_data/cifar-10-batches-py/data_batch_2', 'cifar10_data/cifar-10-batches-py/data_batch_3', 'cifar10_data/cifar-10-batches-py/data_batch_4', 'cifar10_data/cifar-10-batches-py/data_batch_5']
Reading images from cifar10_data/cifar-10-batches-py/data_batch_1
Reading images from cifar10_data/cifar-10-batches-py/data_batch_2
Reading images from cifar10_data/cifar-10-batches-py/data_batch_3
Reading images from cifar10_data/cifar-10-batches-py/data_batch_4
Reading images from cifar10_data/cifar-10-batches-py/data_batch_5
Shuffling
['cifar10_data/cifar-10-batches-py/test_batch']
Reading images from cifar10_data/cifar-10-batches-py/test_batch
Shuffling
WARNING:tensorflow:From /home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /cache/user-job-dir/code/resNeXt.py:56: UniformUnitScaling.__init__ (from tensorflow.python.ops.init_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.initializers.variance_scaling instead with distribution=uniform to get equivalent behavior.
WARNING:tensorflow:From code/cifar10_train.py:276: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
WARNING:tensorflow:From /home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
WARNING:tensorflow:From code/cifar10_train.py:86: all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Please use tf.global_variables instead.
WARNING:tensorflow:From /home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/util/tf_should_use.py:193: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead.
2020-11-06 16:38:53.673816: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA
2020-11-06 16:38:53.855648: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x55a8eacc7940 executing computations on platform CUDA. Devices:
2020-11-06 16:38:53.855689: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): Tesla V100-PCIE-32GB, Compute Capability 7.0
2020-11-06 16:38:53.884534: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3000000000 Hz
2020-11-06 16:38:53.885334: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x55a8eade33b0 executing computations on platform Host. Devices:
2020-11-06 16:38:53.885359: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2020-11-06 16:38:53.885820: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties: 
name: Tesla V100-PCIE-32GB major: 7 minor: 0 memoryClockRate(GHz): 1.38
pciBusID: 0000:21:01.0
totalMemory: 31.75GiB freeMemory: 31.44GiB
2020-11-06 16:38:53.885843: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2020-11-06 16:38:53.887935: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-11-06 16:38:53.887960: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2020-11-06 16:38:53.887967: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2020-11-06 16:38:53.888046: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 30583 MB memory) -> physical GPU (device: 0, name: Tesla V100-PCIE-32GB, pci bus id: 0000:21:01.0, compute capability: 7.0)
2020-11-06 16:38:56.658496: I tensorflow/core/platform/s3/aws_logging.cc:54] Initializing config loader against fileName /home/work//.aws/config and using profilePrefix = 1
2020-11-06 16:38:56.658535: I tensorflow/core/platform/s3/aws_logging.cc:54] Initializing config loader against fileName /run/secrets/kubernetes.io/batch/config and using profilePrefix = 0
2020-11-06 16:38:56.658546: I tensorflow/core/platform/s3/aws_logging.cc:54] Setting provider to read credentials from /run/secrets/kubernetes.io/batch/config for credentials file and /home/work//.aws/config for the config file , for use with profile default
2020-11-06 16:38:56.658558: I tensorflow/core/platform/s3/aws_logging.cc:54] Creating HttpClient with max connections2 and scheme http
2020-11-06 16:38:56.659007: I tensorflow/core/platform/s3/aws_logging.cc:54] Initializing CurlHandleContainer with size 2
2020-11-06 16:38:56.659041: I tensorflow/core/platform/s3/aws_logging.cc:54] Creating Instance with default EC2MetadataClient and refresh rate 900000
2020-11-06 16:38:56.659174: I tensorflow/core/platform/s3/aws_logging.cc:54] Successfully reloaded configuration.
2020-11-06 16:38:56.660612: I tensorflow/core/platform/s3/aws_logging.cc:54] Initializing CurlHandleContainer with size 25
2020-11-06 16:38:56.662755: I tensorflow/core/platform/s3/aws_logging.cc:54] Pool grown by 2
2020-11-06 16:38:56.662772: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.774603: E tensorflow/core/platform/s3/aws_logging.cc:60] No response body. Response code: 403
2020-11-06 16:38:56.774656: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.775728: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.786989: W tensorflow/core/platform/s3/aws_logging.cc:57] Encountered Unknown AWSError
InvalidSecuToken
security token is invalid:
2020-11-06 16:38:56.787023: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.787229: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.793161: E tensorflow/core/platform/s3/aws_logging.cc:60] No response body. Response code: 403
2020-11-06 16:38:56.793194: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.793297: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.805020: W tensorflow/core/platform/s3/aws_logging.cc:57] Encountered Unknown AWSError
InvalidSecuToken
security token is invalid:
2020-11-06 16:38:56.805055: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.805163: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.814266: E tensorflow/core/platform/s3/aws_logging.cc:60] No response body. Response code: 403
2020-11-06 16:38:56.814294: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.814391: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.824878: W tensorflow/core/platform/s3/aws_logging.cc:57] Encountered Unknown AWSError
InvalidSecuToken
security token is invalid:
2020-11-06 16:38:56.824914: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.825037: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.830700: E tensorflow/core/platform/s3/aws_logging.cc:60] No response body. Response code: 403
2020-11-06 16:38:56.830735: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.830854: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.837595: W tensorflow/core/platform/s3/aws_logging.cc:57] Encountered Unknown AWSError
InvalidSecuToken
security token is invalid:
2020-11-06 16:38:56.837627: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.837732: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.847978: E tensorflow/core/platform/s3/aws_logging.cc:60] No response body. Response code: 403
2020-11-06 16:38:56.848007: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.848117: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.895553: W tensorflow/core/platform/s3/aws_logging.cc:57] Encountered Unknown AWSError
InvalidSecuToken
security token is invalid:
2020-11-06 16:38:56.895590: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2020-11-06 16:38:56.895703: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2020-11-06 16:38:56.909727: E tensorflow/core/platform/s3/aws_logging.cc:60] No response body. Response code: 403
2020-11-06 16:38:56.909761: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
Traceback (most recent call last):
  File "code/cifar10_train.py", line 404, in <module>
    train.train()
  File "code/cifar10_train.py", line 99, in train
    summary_writer = tf.summary.FileWriter(train_dir, sess.graph)
  File "/home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/summary/writer/writer.py", line 367, in __init__
    filename_suffix)
  File "/home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/summary/writer/event_file_writer.py", line 67, in __init__
    gfile.MakeDirs(self._logdir)
  File "/home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/lib/io/file_io.py", line 442, in recursive_create_dir
    recursive_create_dir_v2(dirname)
  File "/home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/lib/io/file_io.py", line 458, in recursive_create_dir_v2
    pywrap_tensorflow.RecursivelyCreateDir(compat.as_bytes(path), status)
  File "/home/work/anaconda/lib/python3.6/site-packages/tensorflow/python/framework/errors_impl.py", line 528, in __exit__
    c_api.TF_GetCode(self.status.status))
tensorflow.python.framework.errors_impl.UnknownError: : No response body. Response code: 403
[ModelArts Service Log]modelarts-pipe: total length: 11806
[Modelarts Service Log]Training end with return code: 1
[Modelarts Service Log]Training completed.
